dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -339,6 +338,12 @@ Path findClassLocation(String className) {
     return pathToClasses.build();
   }
 
+  /**
+   * Returns a list of class file names in {@code jar} as in {@link JavaClass#getFileName()}. This
+   * class file name is a path ("." as element separator) that locates a class file in a class path.
+   * Usually class name and class file name are the same for a class. However a class file name may
+   * have a framework-specific prefix. Example: {@code BOOT-INF.classes.com.google.Foo}.
+   */
   static ImmutableSet<String> listClassNamesInJar(Path jar) throws IOException {
```

Comment:
It looks like this method should be named listClassFileNames since it class file names and class names are not the same thing. Also, we probably don't need "InJar". In general, method names don't need to mention their arguments.